ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_of_logical_formulas_and_logical_ontologies/exercises/exercise_3/exercise_3.scs

```diff
@@ -0,0 +1,37 @@
+exercise_3_section_subjdomain_of_logical_formulas_and_logical_ontologies
+<- exercise;
+
+=> nrel_main_idtf:
+ [Упражнение 3. Раздел. Предметная область логических формул и логических онтологий]
+ (* <- lang_ru;; *);
+ [Exercise 3. Section. Subject domain of logical formulas and logical ontologies. ]
+ (* <- lang_en;; *);
+
+<- rrel_key_sc_element: ...
+ (*
+  <- explanation;;
+  <= nrel_sc_text_translation:
+  ...
+  (*
+  -> rrel_example:
+    "file://definition_exercise_3.html"
+   (* <- lang_ru;; *);;
+  *);;
+ *);;
+
+exercise_3_section_subjdomain_of_logical_formulas_and_logical_ontologies
+=> nrel_right_answer: ...
+(*
+<- rrel_key_sc_element: ...
+ (*
+  <- illustration;;
+  <= nrel_sc_text_translation:
+  ...
+  (*
+  -> rrel_example:
+    "file://3.png"  
```

Comment:
I think, you should also name the node that represents the considering logical formula.